extensions/kubernetes/vanilla/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: "Openshift"
+metadata:
+  keywords:
+  - "openshift"
+  guide: "https://quarkus.io/guides/openshift"
```

Comment:
this metadata looks wrong - this is for openshift not kubernetes?

Reply:
Yup, I'll fix it